.github/workflows/e2e.yaml

```diff
@@ -8,6 +8,8 @@ on:
       - 'main'
     tags:
       - 'v*'
+    paths:
+      - '**.go'
```

Comment:
Will a change to the GitHub actions workflow trigger the e2e tests?

Reply:
good catch

Reply:
It should also be on pull_request, not on push